src/test/resources/logging_single_handler.properties

```diff
@@ -1,6 +1,8 @@
 handlers=com.rapid7.jul.LogentriesHandler
 .handlers=com.rapid7.jul.LogentriesHandler
-com.rapid7.jul.LogentriesHandler.token=f2ff093a-54c7-4305-9449-1dccda748347
+
+com.rapid7.jul.LogentriesHandler.token=token-logentries-handler
```

Comment:
Just in case we confuse anyone, can we use valid UUIDs? Not "real" log tokens, but `deadbeef-dead-beef-dead-beefdeadbeef` for example.